pages/api/create-checkout-session.js

```diff
@@ -1,17 +1,15 @@
-import { stripe } from '../../utils/initStripe';
-import { supabaseAdmin } from '../../utils/initSupabaseAdmin';
-import { createOrRetrieveCustomer } from '../../utils/useDatabase';
-import { getURL } from '../../utils/helpers';
+import { stripe } from '@/utils/stripe';
+import { getUser } from '@/utils/supabase-admin';
+import { createOrRetrieveCustomer } from '@/utils/useDatabase';
+import { getURL } from '@/utils/helpers';
 
 const createCheckoutSession = async (req, res) => {
   if (req.method === 'POST') {
     const token = req.headers.token;
     const { price, quantity = 1, metadata = {} } = req.body;
 
     try {
-      const { data: user, error } = await supabaseAdmin.auth.api.getUser(token);
-      if (error) throw error;
```

Comment:
For example, instead of having these checks everywhere, the throwing of errors would be moved to the "library" level inside the helper functions